<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HP-Supernova" ignored="0" name="Functions Test(failed)" tests="6" failures="6" timestamp="2019-12-30T01:05:35 IST" time="136.279" errors="0">
  <testcase name="createAccountOnBewakoof" time="24.307" classname="com.bewkoof.testCases.TC_SignUp_DDT_004">
    <failure type="java.lang.AssertionError" message=" Test Failed Becasue may be mobile number,mailId is already used did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Test Failed Becasue may be mobile number,mailId is already used did not expect to find [true] but found [false]
at com.bewkoof.testCases.TC_SignUp_DDT_004.createAccountOnBewakoof(TC_SignUp_DDT_004.java:91)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- createAccountOnBewakoof -->
  <testcase name="createAccountOnBewakoof" time="23.428" classname="com.bewkoof.testCases.TC_SignUp_DDT_004">
    <failure type="java.lang.AssertionError" message=" Test Failed Becasue may be mobile number,mailId is already used did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Test Failed Becasue may be mobile number,mailId is already used did not expect to find [true] but found [false]
at com.bewkoof.testCases.TC_SignUp_DDT_004.createAccountOnBewakoof(TC_SignUp_DDT_004.java:91)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- createAccountOnBewakoof -->
  <testcase name="createAccountOnBewakoof" time="23.341" classname="com.bewkoof.testCases.TC_SignUp_DDT_004">
    <failure type="java.lang.AssertionError" message=" Test Failed Becasue may be mobile number,mailId is already used did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Test Failed Becasue may be mobile number,mailId is already used did not expect to find [true] but found [false]
at com.bewkoof.testCases.TC_SignUp_DDT_004.createAccountOnBewakoof(TC_SignUp_DDT_004.java:91)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- createAccountOnBewakoof -->
  <testcase name="loginIntoBewakoof" time="12.414" classname="com.bewkoof.testCases.TC_LoginDDT_003">
    <failure type="java.lang.AssertionError" message=" Test Failed Because Either Email Or Password is Wrong(Invalid Credentials did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Test Failed Because Either Email Or Password is Wrong(Invalid Credentials did not expect to find [true] but found [false]
at com.bewkoof.testCases.TC_LoginDDT_003.loginIntoBewakoof(TC_LoginDDT_003.java:53)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginIntoBewakoof -->
  <testcase name="loginIntoBewakoof" time="12.382" classname="com.bewkoof.testCases.TC_LoginDDT_003">
    <failure type="java.lang.AssertionError" message=" Test Failed Because Either Email Or Password is Wrong(Invalid Credentials did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Test Failed Because Either Email Or Password is Wrong(Invalid Credentials did not expect to find [true] but found [false]
at com.bewkoof.testCases.TC_LoginDDT_003.loginIntoBewakoof(TC_LoginDDT_003.java:53)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginIntoBewakoof -->
  <testcase name="loginIntoBewakoof" time="12.086" classname="com.bewkoof.testCases.TC_LoginDDT_003">
    <failure type="java.lang.AssertionError" message=" Test Failed Because Either Email Or Password is Wrong(Invalid Credentials did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError:  Test Failed Because Either Email Or Password is Wrong(Invalid Credentials did not expect to find [true] but found [false]
at com.bewkoof.testCases.TC_LoginDDT_003.loginIntoBewakoof(TC_LoginDDT_003.java:53)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- loginIntoBewakoof -->
</testsuite> <!-- Functions Test(failed) -->
